04_numeronym/alextmz/main_test.go

```diff
@@ -0,0 +1,63 @@
+package main
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNumeronym(t *testing.T) {
+	var tests = map[string]struct {
+		arg  string
+		want string
+	}{
+		"empty string":               {"", ""},
+		"single string of 1 element": {"a", "a"},
```

Comment:
Your test cases are a little lacking - you really should be testing boundary conditions, so you want a test case of length 3 and 4. You should also have some test cases at the boundary with non-ascii characters.

I suggest you add a test case for "💩", "💩💩", "💩💩💩" and "💩💩💩💩" perhaps (or any non-ascii rune - for some childish reason, pile-of-poo is my go-to non-ascii rune). If you add those, you may see some interesting results: https://goplay.space/#aaUgdA0erhN

Reply:
Tests added